### PR TITLE
added ability to install ST2 packs

### PR DESCRIPTION
**Pull request by guymatz**
set st2_packs to have them installed.  This should be a module, but this is good enough for now

### CODE REVIEW

**Comment by armab** [comment]
In a meantime, don't worry about CI, - they're not related to your change, something new from upstreams.
I'll look at them in another PR.

File: roles/StackStorm.st2/tasks/packs.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+- name: Get installed st2 packs (in json format)
+  command: /usr/bin/st2 pack list -j
```
Comment (armab) [inline roles/StackStorm.st2/tasks/packs.yml]: CI is failing here: https://travis-ci.org/StackStorm/ansible-st2/jobs/498299271#L1288 with the following:
```
fatal: [localhost]: FAILED! => {"changed": false, "cmd": ["/usr/bin/st2", "pack", "list", "-j"], "delta": "0:00:00.356562", "end": "2019-02-25 18:25:31.978348", "msg": "non-zero return code", "rc": 1, "start": "2019-02-25 18:25:31.621786", "stderr": "", "stderr_lines": [], "stdout": "ERROR: HTTPConnectionPool(host='127.0.0.1', port=9101): Max retries exceeded with url: /v1/packs?include_attributes=ref%2Cname%2Cdescription%2Cversion%2Cauthor (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7ffb2fd1e290>: Failed to establish a new connection: [Errno 111] Connection refused',))", "stdout_lines": ["ERROR: HTTPConnectionPool(host='127.0.0.1', port=9101): Max retries exceeded with url: /v1/packs?include_attributes=ref%2Cname%2Cdescription%2Cversion%2Cauthor (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7ffb2fd1e290>: Failed to establish a new connection: [Errno 111] Connection refused',))"]}
```
Which means StackStorm is not running/started yet at the moment we're trying to run pack install.

Sounds like this functionality needs to be moved to a later post-services start step.

**Comment by armab** [comment]
Just fixed some upstream linting CI issues, not related to your PR.

Now CI is signalizing real failures coming from this PR. I advised some changes and improvements above ^^ so we can merge it.
@guymatz Let us know if you're interested/have capacity to finish this contribution.

**Comment by guymatz** [comment]
Hi!  I would like to put something like this in smoketests:
- name: verify install of packs
  command: st2 pack get st2
  changed_when: false
  tags:
    - smoke-tests

but it looks like there's a task in smoktests to install the st2 pack . .
.  is it still a valid test of the playbook?

**Comment by armab** [comment]
Yes. We can install `st2` pack by default by having `st2_packs: [st2]` in Ansible defaults and check if it's present during the smoke tests, as you wanted.

Having `st2` pack installed in default installation is something that we follow in other deployment methods so this approach should be consistent.

**Pull request closed without merging**